src/common/Types.ts

```diff
@@ -938,7 +938,7 @@ export interface PopupOptions {
     // anchor has been pressed.
     // IMPORTANT NOTE: This handler may be called when the component is
     // already unmounted as it uses a time delay to accommodate a fade-out animation.
-    onAnchorPressed?: (e: RX.Types.SyntheticEvent) => void;
+    onAnchorPressed?: (e?: RX.Types.SyntheticEvent) => void;
```

Comment:
Why is this parameter optional? Seems like it should be required.

Reply:
talked offline